enterprise/server/hostedrunner/hostedrunner.go

```diff
@@ -45,6 +45,8 @@ const (
 	// This is used by default.
 	nonRootUser = "buildbuddy"
 	rootUser    = "root"
+
+	timeoutGracePeriod = 10 * time.Second
```

Comment:
(fyi) I think the caveats mentioned on this same constant in the workflow service package still apply. So if it takes longer than 10s to start the CI runner, it won't have enough time to finalize the invocation BES stream and it'll wind up in "disconnected" / "in-progress" state.

I guess at some point we probably want the executor to be in charge of setting the timeout, and maybe set it as a deadline timestamp rather than a relative timeout.